numpy/distutils/misc_util.py

```diff
@@ -2326,6 +2326,41 @@ def get_info(name):
                 return g.get(name, g.get(name + "_info", {}))
 
             def show():
+                """
+                Print information about various resources (libraries, library
+                directories, include directories, etc.) in the system on which
+                NumPy was built.
+
+                See Also
+                --------
+                get_include : Returns the directory containing NumPy C
```

Comment:
I think usually see also comes last.

Reply:
`Examples` is the last section: https://numpydoc.readthedocs.io/en/latest/format.html#sections